components/rendering/src/shortcode.rs

```diff
@@ -411,4 +414,20 @@ Some body {{ hello() }}{%/* end */%}"#,
         let res = render_shortcodes("Body\n {% youtube() %}\nHello \n World{% end %}", &tera);
         assert_eq!(res, "Body\n Hello \n World");
     }
+
+    #[test]
+    fn outer_newlines_removed_from_shortcodes_with_body() {
```

Comment:
the titles from those tests doesn't really seem to match what they are doing no? They strip outer newlines whitespaces, not the newlines themselves

Reply:
I couldn't think of a concise way to describe the behavior as a test name.

It removes all outer newlines (and the whitespace between them), but the parenthetical was, to my mind, just a necessary byproduct.

If you have better test names in mind I'll gladly change them.